pkg/math/oracle/median.go

```diff
@@ -38,11 +38,6 @@ func NewMedianAggregator(
 		return nil, err
 	}
 
-	if cfg.AggregationType != mmtypes.AggregationType_INDEX_PRICE_AGGREGATION {
```

Comment:
Is there a reason we can't remove the agg-type from the config, now?

Reply:
I left it since we're rewriting all of the mm stuff and deleting all the proto and tests didn't seem worth it at this point.

Reply:
What is the reason we need to remove this rn? From my perspective I put this in the case where operators accidentally use the wrong market.json file. this was an easy way to make sure things were configured correctly. the dYdX market params API constructs the market.json to include the field to be set correctly.

Reply:
Supporting multiple aggregation types seems like a really bad idea in general. It will make it even harder to reason about how a price was determined and introduce potential for multi-modal behavior in our on chain prices based on a bug in one place but not another.

I also removed it because it was the one stateful piece of validation that was preventing us from bootstrapping the oracle without a `market.json` config.

Reply:
yea agree, multiple not gud. im fine making this change now